### ISSUE TITLE
ChangeLog should update

### ISSUE THREAD

**Issue by TrumanLing** [opencontainers/image-spec#326]
version 0.5.0 has been released for two days since Sep. 17th, does it need to update the change log?

**Comment by wking** [comment]
On Sun, Sep 18, 2016 at 08:26:06PM -0700, TrumanLing wrote:
> version 0.5.0 has been release for two days since Sep. 17th, does it
> need to update the change log?

I think that's just waiting on someone writing up the entries and
submitting a PR [1].  It would be nice to have that ChangeLog be part
of the release criteria [2], but we don't have official release
criteria yet.  If you get @vbatts ChangeLog entries before he gets two
LGTMs he still needs in #289 [3], you can probably get them in with
that branch ;).

[1]: https://github.com/opencontainers/image-spec/pull/289#issuecomment-246803009
[2]: https://github.com/opencontainers/tob/pull/15#issuecomment-225988785
[3]: https://groups.google.com/a/opencontainers.org/d/msg/dev/71PyKIBH3os/BDdHHULqAwAJ

**Comment by vbatts** [comment]
On 18/09/16 20:26 -0700, TrumanLing wrote:
>version 0.5.0 has been release for two days since Sep. 17th, does it need to update the change log?

Happily i'll take an update to ChangeLog. It may likely just roll into
the next release, and entries for v0.5.0 will just be historical.

**Comment by vbatts** [comment]
@trumanling I agree it's nice to have a ChangeLog. I would let it be a release blocker, if we _did_ have an automation for producing it. As it is, making the ChangeLog is just best effort.
I welcome PRs to update the ChangeLog. Perhaps if folks do it incrementally, that would be more manageable. I'm inclined to close this issue and just encourage PRs from folks.

**Comment by stevvooe** [comment]
@vbatts Anything actionable here or can we close this?

**Comment by vbatts** [comment]
@stevvooe it's still a nice-to-have, but not something that is a formal release step.
